src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocCustomTag.java

```diff
@@ -1,8 +1,10 @@
-package com.puppycrawl.tools.checkstyle.checks.javadoc.abstractjavadoc;
-
 /*
- * Config: TempCheck
- */
+AbstractJavadocCheckTest$Temp
```

Comment:
such Inputs should not have such comment at all.
AbstractJavadoc is not real Check, it has not instances, and nobody use them outside of Checkstyle.

BDD is behavior based testing concept that is focused on user/human.
One more time " ..... focused on human".

What we have here is just another example of pure UTs that create synthetic test cases to cover some code just for code coverage satisfaction , without any benefit to real life usages or real code coverage.

Ideally we should be able to cover all code in final Checks and in all classes that Checks extend by means of execution on real Checks. So no TempCheck, no DummyCheck, ... .

If that is not possible, there definetely cases like this, is this case we do not need BDD ideas and we will keep pure UTs methods and it will be tests to just satisfy coverage in VERY edge cases.

Lets avoid updates for such edge cases, we will come back to them at the very end, and we will try remove AbstractJavadocCheckTest.java  completely and extend test cases in real Checks to improve coverage to fanatic 100%